cluster-autoscaler/cloudprovider/openshiftmachineapi/machineapi_controller_test.go

```diff
@@ -761,17 +769,17 @@ func TestControllerNodeGroupsNodeCount(t *testing.T) {
 		if err != nil {
 			t.Fatalf("unexpected error: %v", err)
 		}
-		if got := len(nodegroups); got != tc.nodeGroups {
-			t.Fatalf("expected %d, got %d", tc.nodeGroups, got)
+		if got := len(nodegroups); got != tc.expectedNodeGroups {
+			t.Fatalf("expected %d, got %d", tc.expectedNodeGroups, got)
 		}
 
 		for i := range nodegroups {
 			nodes, err := nodegroups[i].Nodes()
 			if err != nil {
 				t.Fatalf("unexpected error: %v", err)
 			}
-			if got := len(nodes); got != tc.nodesPerGroup {
-				t.Fatalf("expected %d, got %d", tc.nodesPerGroup, got)
+			if got := len(nodes); got != tc.expectedNodesPerGroup {
+				t.Fatalf("expected %d, got %d", tc.expectedNodesPerGroup, got)
 			}
 		}
 	}
```

Comment:
Why passing `tc.nodesPerGroup, int32(tc.nodesPerGroup)`  instead of just passing nodesPerGroup once. Are these two values unequal ever?

Reply:
For some tests, yes. However, as I mentioned in a previous comment that I think these can now be collapsed. The case, and I think there's only one, where we need these to be different can be altered in that test after the config has been created.

Reply:
> For some tests, yes. However, as I mentioned in a previous comment that I think these can now be collapsed. The case, and I think there's only one, where we need these to be different can be altered in that test after the config has been created.

https://github.com/openshift/kubernetes-autoscaler/pull/106